### PR TITLE
Included info on Microsoft Repository

### PR DESCRIPTION
**Pull request by ewhitesides**
https://technet.microsoft.com/en-us/windows-server-docs/compute/linux-package-repository-for-microsoft-software?f=255&MSPPError=-2147217396

### CODE REVIEW

File: docs/installation/linux.md [hunk 1]
```diff
@@ -92,6 +92,11 @@ You can also install the RPM without the intermediate step of downloading it:
 ```sh
 sudo yum install https://github.com/PowerShell/PowerShell/releases/download/v6.0.0-alpha.17/powershell-6.0.0_alpha.17-1.el7.centos.x86_64.rpm
 ```
+You can also install via the Microsoft Repository
+sudo rpm -Uvh http://packages.microsoft.com/config/rhel/7/packages-microsoft-prod.rpm
```
Comment (TravisEz13) [inline docs/installation/linux.md]: This should have code blocks.  I can add them.  Just ping me if you want me to do it.

**Review by TravisEz13 — CHANGES_REQUESTED** [review]

File: docs/installation/linux.md (hunk 1 above)
Comment (ewhitesides) [inline docs/installation/linux.md]: You can go ahead... still getting used to how github works.  Thanks!

**Comment by joeyaiello** [comment]
First, thanks for this contribution, we desperately needed to put these instructions in docs. 

I went ahead and made a few ordering changes to showcase the repository installation method as the preferred method. However, it looks like another PR came in and did the same before this one got merged. I'm going to fix the merge conflict here as well.

**Review by joeyaiello — APPROVED** [review]

**Comment by joeyaiello** [comment]
Alright, I made a few more wording changes when fixing the merge commit. I still need to do a pass here at some point on the whole doc, but it's absolutely in a better place than it was a week ago. :smile:

**Review by TravisEz13 — APPROVED** [review]

**Pull request merged**